_shows/17_18/do_i_wanna_know.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Do I Wanna Know
+playwright: Felicity Chilver
+student_written: true
+period: Spring
+season: Fringe
+season_sort: 220
+date_start: 2018-03-18
+date_end: 2018-03-19
+venue: New Theatre Studio A
+
+cast:
+- role: James
+  name: Ted Marriott
+- role: Evelyn
+  name: Sophie Walton
+
+crew:
+- role: Director
+  name: Felicity Chilver
+- role: Producer
+  name: Sam Osborne
+- role: Assistant Director
+  name: Georgina Pittman
+- role: Technical Director
+  name: Laura Wolczyk
+- role: Lighting Designer
+  name: Nathan Penney
+- role: Sound Designer
+  name: James Curling
+- role: Poster Design
+  name: Felicity Chilver
+- role: Photographer
+  name: Nathan Penney
```

Comment:
Can you add Marsya Hazanan as Stash Designer please?

Reply:
done